test/generators/java/__snapshots__/JavaGenerator.spec.ts.snap

```diff
@@ -583,7 +842,26 @@ exports[`JavaGenerator should render \`class\` type 1`] = `
 
   public Map<String, Object> getAdditionalProperties() { return this.additionalProperties; }
   public void setAdditionalProperties(Map<String, Object> additionalProperties) { this.additionalProperties = additionalProperties; }
-}"
+}",
+  "/**
+ * Members represents a union of types: String, Double, Boolean
+ */
+public interface Members {
+  
+}",
```

Comment:
These "stub" types representing unions that we aren't using aren't great. I can't find an idiomatic way to suppress them and they would be used if we extended the `oneOf` support to include built-in `Java` types in the future. They don't do much harm but they represent a bit of a loose end for this feature.

Reply:
I am guessing that the limits to the Java generator at the moment is simple types cannot be represented in unions i.e. string + boolean model, right? i.e. only one or more classes

Reply:
Java can't include built in types in a new interface. We could wrap these types in a new class but that isn't great either. My plan is to get this all working for user defined classes and fall back to `Object` for built ins. If we want to have the option of using built in types, this could be a follow on PR.

Reply:
Just remember that even though unions are a non-thing in Java, it's still something that users can define within inputs, and because of that we must represent it in the best possible way.

> We could wrap these types in a new class but that isn't great either

I have a feeling this is exactly the best solution that encompasses all use-cases, and is rather simple to achieve. But lets see 😄

Reply:
I'm prioritising composable smaller PRs over one giant PR. I don't have an issues with doing the type wrapping, but I'm trying to keep each PR to a manageable size.